src/main/java/org/opendatakit/suitcase/ui/PullPanel.java

```diff
@@ -65,11 +72,22 @@ public PullPanel(IOPanel parent) {
         gbc.weighty = 2;
         this.add(pullInputPanel, gbc);
 
+        JButton logoutButton = new JButton();
+        logoutButton.setText(LOGOUT);
+        logoutButton.setBorder(BorderFactory.createEmptyBorder(10,10,10,10));
+        logoutButton.addActionListener(new ActionListener() {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+              logout();
+            }
+        });
+        gbc.weighty = 1;
+        this.add(logoutButton);
```

Comment:
The logout button construction should be its own method, so it's easier to read.